stdlib/tkinter/__init__.pyi

```diff
@@ -671,8 +692,18 @@ class Wm:
     def wm_attributes(self) -> tuple[Any, ...]: ...
```

Comment:
IMO this first "no arguments" overload should only be applied on Python versions older than 3.13. On 3.13 and newer, you shouldn't get these two overloads, because the first one is unnecessary:
- `wm_attributes(self)`
- `wm_attributes(self, *, return_python_dict: Literal[False] = False)`